### DISCUSSION TITLE
LPS Node ID cannot be changed.

### DISCUSSION THREAD

**Discussion by seongyoeng** [bitcraze/discussions#558] [Q&A]
Firmware updates to LPS nodes are successful However, you cannot change the LPS node ID.
I tried to change id using LPS tools, but the Apply button is not activated. What should I do in this case?

**Comment by krichardsson** [comment]
The node should **not** be in DFU mode for the configuration part of the LPS tools to work. Rebooting the node without holding any button will put it in the "normal" mode and after this it should be possible to change the configuration from the LPS tools. 

When the node is booted in "normal" mode, it will appear as a serial port on USB and it is possible to communicate with it using any terminal program. It is possible to set the id this way as well, please see [the low level config docs](https://www.bitcraze.io/documentation/repository/lps-node-firmware/master/development/anchor-low-level-config/). You can try this if you still have problems with the LPS tool.

**Reply by CalvinKoung** [reply]
hi, the node in the normal mode but apply button still cannot activate. Can you help me to solve ? thanks.

**Reply by ataffanel** [reply]
What operating system are you using and how did you install the lps tools?

**Reply by CalvinKoung** [reply]
I use window system and follow the bitcraze website instruction. But finally I use two computer to solve this problem. I update in one computer and set the node ID in another computer.

**Reply by ataffanel** [reply]
Oh I think I understand what is happening then: you likely installed the libusb driver using zadig for the running firmware as well, not only for the bootloader.

Zadig should only be used to install the driver for the bootloader. The firmware is using the windows-provided Serial port driver.

The way to fix the first computer is to open Zadig when one node is connected, check the box to see all devices in one of the menu, and install the "usbser" driver for the node. This way the node will be back to be seen as a serial port and everything will work on the first computer.
